### ISSUE TITLE
properties are create multiple times

### ISSUE THREAD

**Issue by guustnieuwenhuis** [tpryan/Apptacular#30]
I get the error "Properties cannot be declared more than once" when using the code generated by Apptacular.

When I take a look in the code I see that on several location, properties are created twice.

**Comment by tpryan** [comment]
Without more info I cannot reasonably troubleshoot. Can you post a schema to go with.

**Comment by guustnieuwenhuis** [comment]
I've reproduced the issue on a clean project. This is what I've done:
1. create new CFBuilder project
2. create Apptacular Application
3. edit database model > change entity name on one of the tables
4. regenerate Application

Now the entity for witch you just changed the entity name has double properties.

I've had to change the entity name because the name of the column is "member" and this is a reserved word in ColdFusion ORM/Hibernate. I changed it to iMember (not because of Apple, the first letter of the name of the application is "i").

I've send you the schema in december per email, let me know if you want me to resend it.
